pyanaconda/ui/lib/space.py

```diff
@@ -77,7 +79,9 @@ def check(self):
         log.info("fs space: %s  needed: %s", free, needed)
         self.success = (free > needed)
         if not self.success:
-            self.deficit = needed - free
+            required_space = Size(Decimal(needed) / Decimal(self.storage.rootDevice.format.MetadataSizeFactor))
```

Comment:
Should these two really be divided? It doesn't make much sense to me. Also, this calculation should be put into a method with a reasonable name.